<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProgrammerWindow</class>
 <widget class="QWidget" name="ProgrammerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>661</width>
    <height>285</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ProgrammerWindow</string>
  </property>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>481</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="revisedLabel">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>80</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Revised</string>
   </property>
  </widget>
  <widget class="QLabel" name="unrevisedLabel">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>130</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Unrevised</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="fileLineEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>481</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="addButton">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>230</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QPushButton" name="reviseButton">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>30</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Revise</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
